remme/clients/block_info.py

```diff
@@ -107,7 +127,7 @@ def create_block_address(block_num):
     @staticmethod
     def interpret_block_info(block_info, votes):
         return {
-            "block_number": block_info.block_num + 1,
+            "block_number": block_info.block_num,
```

Comment:
Then it will show u zero block

Reply:
yes, that will be correspond to data in this dictionary. Because if we will do + 1 so data doesn't correspond to block_num